### DISCUSSION TITLE
OBS Studio 28.01 Mac update

### DISCUSSION THREAD

**Discussion by leafygh** [obsproject/obs-studio#7277] [General]
The update announces itself, downloads and extracts. But then it gets stuck on installing and we have to quit OBS without the update completing.

**Comment by fzwoch** [comment]
I believe I have seen the same when updating from RC1/2 to the final 28.0.0 as well.

**Reply by derrod** [reply]
Yes we've noticed that - at least on ARM - there appear to be issues with Qt/Sparkle (in my case it crashes after clicking install). It's noted but we haven't gotten around to investigating it properly yet.

**Comment by RytoEX** [comment]
We've opened PR #7296 to address this issue.

**Reply by leafygh** [reply]
Thank you, RytoEX

**Comment by leafygh** [comment]
Just a side issue:

Why are your emails using Cc: instead of the much more prudent Bcc:

Surely Bcc: should be adopted as standard policy?

**Reply by RytoEX** [reply]
We have no control over emails sent by GitHub and how they are configured. If you have feedback on that, please contact GitHub.
